scylla/src/response/rows_displayer.rs

```diff
@@ -0,0 +1,1191 @@
+        write_colored!(
+            f,
+            self.settings.print_in_color,
+            self.settings.colors.int,
+            "{}",
+            self.value.0
+        )
+    }
+}
+
+// float
+impl fmt::Display for WrapperDisplay<'_, f32> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.settings.exponent_displaying_floats {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.float,
+                "{:e}",
+                self.value
+            )
+        } else {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.float,
+                "{}",
+                self.value
+            )
+        }
+    }
+}
+
+// double
+impl fmt::Display for WrapperDisplay<'_, f64> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.settings.exponent_displaying_floats {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.float,
+                "{:e}",
+                self.value
+            )
+        } else {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.float,
+                "{:.digits$}",
+                self.value,
+                digits = self.settings.double_precision
+            )
+        }
+    }
+}
+
+// blob
+impl fmt::Display for WrapperDisplay<'_, Vec<u8>> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.settings.byte_displaying == ByteDisplaying::Hex {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.blob,
+                "0x"
+            )?;
+        }
+        for byte in self.value {
+            match self.settings.byte_displaying {
+                ByteDisplaying::Ascii => {
+                    write_colored!(
+                        f,
+                        self.settings.print_in_color,
+                        self.settings.colors.blob,
+                        "{}",
+                        *byte as char
+                    )?;
+                }
+                ByteDisplaying::Hex => {
+                    write_colored!(
+                        f,
+                        self.settings.print_in_color,
+                        self.settings.colors.blob,
+                        "{:02x}",
+                        byte
+                    )?;
+                }
+                ByteDisplaying::Dec => {
+                    write_colored!(
+                        f,
+                        self.settings.print_in_color,
+                        self.settings.colors.blob,
+                        "{}",
+                        byte
+                    )?;
+                }
```

Comment:
This will write each byte as decimal, and concatenate this. 
This will result in a very unreadable output. Is that what cqlsh does?
Even if it is, we should do it in some actually usable way.